frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+   And user can view knative service are rectangular shaped with round corners
+   And user can see dotted boundary with text "No Revision" mentioned
+   And knative sevice app can have application url on top-right of the node
+   And user sees build decorator on bottom left on knative service app which will take user to build tab
+   Then user checks knative service having label "KSVC" and then the name of service
+
+@regression, @smoke
+Scenario: Visual for knative service with revisions
+   Given user is in topology with knative workload
+   When user checks nodes and the decorators associated with them
+   And user can view knative service are rectangular shaped with round corners
+   And user can see knative service app with dotted boundary with revision present inside it
+   And knative sevice app can have application url on top-right of the node
+   And user can see traffic distribution from knative sevice app to its revisions with it's percentage number
+   And pod ring associated with revisions are present around node with color according to the pod status
+   And knative revision can have application url on top-right of the node 
+   And user sees edit source code decorator is on bottom right of the revision which can lead to github or che workspace
+   And user sees build decorator on bottom left on knative service app which will take user to either build tab 
+   And user checks revisions having label "REV" and then the name
+   Then user checks knative service having label "KSVC" and then the name of service
+
+@regression, @smoke
+Scenario: Sidebar of worload
+   Given user is in the topology with workloads
+   When user clicks on workload
+   And right sidebar opens with Resources tab selected by default 
+   And user checks for sidebar tabs as Details, Resources and Monitoring
+   And user verifies name of the node and Action menu present on top of the sidebar
+   And user may sees health check notifiation above the tabs for deployment and deployment-config
+   Then user checks for close button on top right corner of sidebar 
+   
+@regression, @smoke
+Scenario: Create visual connection between two nodes
+   Given user is in the topology with three nodes
+   When user opens sidebar of one of the node
+   And user opens action menu and selects "Edit Annotations" option
+   And user enters key vlue as "app.openshift.io/connects-to" and value as name of the node to which it will be associated
+   And user can see the arrow connecting them with head pointing to the node whose value is provided in the "Edit Annotations"
+   And user click on the front of arrow and drag it on to the third node and drop it
+   Then user can see the arrow connecting them with head pointing to the node where the arrow is dropped
+
+@regression, @smoke
+Scenario: context menu of nodes
+   Given user is in the topology
+   When user right clicks on the node 
+   And user sees context menu
+   Then user checks options
```

Comment:
Can you please add the options?

Reply:
It would be a long list so it is better to check them by names them while performing any actions with them like editing annotations, adding label etc.